### PR TITLE
feat(*): refactor p2p module to adapt the function of adding and deleting nodes in RBFT

### PR DESCRIPTION
**Pull request by Lizen0512**

### CODE REVIEW

File: cmd/bitxhub/client/node.go [hunk 1]
```diff
@@ -39,3 +39,30 @@ func getValidators(ctx *cli.Context) error {
...
+	if ctx.NArg() < 1 {
+		return fmt.Errorf("please input pid")
+	}
+
+	url, err := getURL(ctx, "delvpnode/"+ctx.Args().Get(0))
+	if err != nil {
+		return err
+	}
+
+	data, err := httpGet(url)
```
Comment (sandyzhou) [inline cmd/bitxhub/client/node.go:61]: Maybe more reasonable to use "post" to delete a node?

File: pkg/peermgr/notifiee.go [hunk 2]
```diff
@@ -1,29 +1,67 @@
...
+	for _, p := range peers {
+		if p.Pid == newAddr {
+			return
+		}
+	}
+	if n.newPeer == "" {
+		n.newPeer = newAddr
+		n.logger.Infof("Updating notifiee newPeer %s", newAddr)
+	}
+	n.logger.Infof("The newPeer %s is not nil, skip the new addr %s", n.newPeer, newAddr)
```
Comment (sandyzhou) [inline pkg/peermgr/notifiee.go:45]: Should it be in "else" branch?

File: cmd/bitxhub/client/node.go (hunk 1 above)
Comment (Lizen0512) [reply cmd/bitxhub/client/node.go:61]: fixed

**Review by Alexader — APPROVED** [review]

File: pkg/peermgr/notifiee.go (hunk 2 above)
Comment (Lizen0512) [reply pkg/peermgr/notifiee.go:45]: fixed

**Review by sandyzhou — APPROVED** [review]

**Pull request merged**